### DISCUSSION TITLE
Rosbag play. "ROS 2 .db3 files do not contain message definitions"

### DISCUSSION THREAD

**Discussion by Enkrl-last** [foxglove/community#944] [General]
I come across a warning about "message definitions" when playing rosbag.
I am aware of the possibility of translation.db3 is in mcap format, but I would not like to do this. I also have access to message definitions in .msg files. Can I export them directly or create an extension that allows you to export schemas when using foxgloves?

**Comment by defunctzombie** [comment]
There is no way to currently load .db3 files and custom schemas. We currently require mcap files for this functionality.
